### PR TITLE
feat: adding cards for team memeber details and implementing search

### PR DESCRIPTION
**Pull request by SharmilaNarendra**
1. Added cards with below details of all the team members and implementated search on the below fields:

- name
- place
- company
- designation

Application link:
https://namaste-react-gb57jdq94-sharmilanarendra.vercel.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,44 +2,59 @@ import React from "react";
 import ReactDOM from "react-dom/client";
```
Comment (ap221882) [inline src/App.js:2]: Leave a blank line between global and local imports

File: src/App.js [hunk 2]
```diff
@@ -2,44 +2,59 @@ import React from "react";
...
-/** Accessing DOM to render react element*/
+const CardContainer = () => {
+    const cards = data.map((member) =>{
+        return  <CardComponent members={member}/>
+    })
+   return cards;
+}
+
+/** body component */
+const BodyComponent = ()=>{
```
Comment (ap221882) [inline src/App.js]: If making it a component not a container, better to make it like -
``` 
const BodyComponent = ({children}) => {
    return (
    <div class="cardContainer">
    {children}
    </div>
    )
}
```

Then wrap where you want to render this component as 
```
<BodyComponent><CardContainer></BodyComponent>
```

File: src/App.js [hunk 3]
```diff
@@ -2,44 +2,59 @@ import React from "react";
...
         <form className="search-bar">
-            <input className="input-search" type="search" placeholder="Search the Hustler.." />
+            <input className="input-search" type="search" placeholder="Search the Hustler..." />
             <button type="submit"><i className="fa fa-search fa-2x"></i></button>
         </form>
     )
 };
 
-/** Header */
+/** header component*/
```
Comment (ap221882) [inline src/App.js:12]: you can separate your components and containers in different files.
make folder structure like

/containers
   //files

/components
  //files

OR

/components
   //file
      ///file.jsx
      ///file.css

File: src/CardComponent.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/** display user information in a card */
+const CardComponent = ({members}) =>{
```
Comment (ap221882) [inline src/CardComponent.js]: You can destructure here only like

{members:{img,name,city,company}}

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,8 @@
+.app-body{
```
Comment (ap221882) [inline src/style.css]: there will be only one body tag in your HTML so can use css HTML tag selector, directly here.
body{
}

File: src/utils/Constants.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+const title = "HUSTLERS";
```
Comment (ap221882) [inline src/utils/Constants.js]: only name your component files as PascalCase, avoids confusion

File: src/components/SearchBar.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import {useState} from "react";
+import data from "../utils/data.json"
+
+
+/** search functionality to search memeber name and designation */
+const SearchMembers = (searchText) =>{
```
Comment (ap221882) [inline src/components/SearchBar.js]: You can make this a utility function and can add it to the utils folder. Also, name it in lowercase.

**Review by ap221882 — COMMENTED** [review]
Well styled. Added some suggestions.

**Review by ap221882 — COMMENTED** [review]
These changes I've suggested for assignments 5 and 6 in total.

File: src/CardComponent.js (hunk 4 above)
Comment (Pushkarkaushik1997) [reply src/CardComponent.js]: This suggestion is really good.

**Pull request merged**